internal/pkg/manifest/lb_web_svc.go

```diff
@@ -46,6 +46,7 @@ type LoadBalancedWebService struct {
 
 // LoadBalancedWebServiceConfig holds the configuration for a load balanced web service.
 type LoadBalancedWebServiceConfig struct {
+	Domain      *string
```

Comment:
hmm this is v interesting 🤔 It's the first time where we are introducing a field that cannot be configured by users.

What do you think of changing it like this and moving it to the bottom of the struct:
```suggestion
	// Fields that are used while marshaling the template for additional clarifications,
	// but don't correspond to a field in the manifests.
	AppDomain      *string
```
Basically separating the fields into two: ones that can be configured by the user and ones that can't.